packages/css-library/src/tokens/flexbox.scss

```diff
@@ -0,0 +1,95 @@
+@use './_variables' as *;
+
+$tokens-flex: (
+  1: (
+    slug: '1',
+    content: 1 0 0%,
+    extend: (
+      'flex-grow': 1,
+    ),
+  ),
```

Comment:
Looks like we might not need to specify flex-grow [according to the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/flex).

> ```css
>    /* Three values: flex-grow | flex-shrink | flex-basis */
>    flex: 2 2 10%;
>  ```
Our [current approach seems redundant](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/a831cc59059d2ef01c161f03e4fdb3caa5c5a8d4/packages/formation/sass/utilities/_flexbox.scss#L38) since we specify `1` as the first part of the base `flex` property.

Reply:
The flex-grow property has been removed from the flex utility classes.